libraries/protocol/include/steem/protocol/config.hpp

```diff
@@ -113,7 +113,8 @@
 #define STEEM_VOTE_DUST_THRESHOLD             (50000000)
 
 #define STEEM_MIN_ROOT_COMMENT_INTERVAL       (fc::seconds(60*5)) // 5 minutes
-#define STEEM_MIN_REPLY_INTERVAL              (fc::seconds(20)) // 20 seconds
+#define STEEM_MIN_REPLY_INTERVAL_PRE_HF20     (fc::seconds(20))   // 20 seconds
```

Comment:
The naming scheme we have used for such changes is to name when a particular constant was introduced. See `STEEM_MAX_VOTED_WITNESSES` as an example.

Here, the names should be `STEEM_MIN_REPLY_INTERVAL_HF0` and `STEEM_MIN_REPLY_INTERVAL_HF20`.